csrc/mmdeploy/apis/c/CMakeLists.txt

```diff
@@ -80,5 +80,8 @@ if (MMDEPLOY_BUILD_SDK_CSHARP_API OR MMDEPLOY_BUILD_SDK_MONOLITHIC)
     set_target_properties(mmdeploy PROPERTIES
             VERSION ${MMDEPLOY_VERSION}
             SOVERSION ${MMDEPLOY_VERSION_MAJOR})
+    set_target_properties(mmdeploy PROPERTIES
+            INSTALL_RPATH "\$ORIGIN"
```

Comment:
According to https://gitlab.kitware.com/cmake/community/-/wikis/doc/cmake/RPATH-handling, this may not work for macOS